src/extract.js

```diff
@@ -574,7 +566,8 @@ exports.Extractor = class Extractor {
       .filter((x) => x !== undefined);
   }
 
-  _extractTranslationData(filename, content) {
+  _extractTranslationData(filename, filecontent) {
+    let content = filecontent;
```

Comment:
I don't understand why this whole renaming of `content` to `filecontent` is necessary, especially if you rename it to `content` at the very end ^^ 

Either finish the job and rename everything to `fileContent` in a separate commit (paying attention to the camelCase convention that exists in this package) or don't do it ?

Reply:
I honestly didn't remember (it's been a while). The original code violates the `no-param-reassign` (and `no-shadow` on line 276) eslint rule.

I'll rename it to fileContent.

Reply:
Oh, that's interesting.